src/josepy/util.py

```diff
@@ -131,7 +136,7 @@ def __hash__(self):
 
 
 class ComparableECKey(ComparableKey):  # pylint: disable=too-few-public-methods
-    """Wrapper for ``cryptography`` RSA keys.
+    """Wrapper for ``cryptography`` EC keys.
```

Comment:
Will also revert this. There are several minor typos related to the EC key support in josepy.